csi/cmd/block/Dockerfile

```diff
@@ -12,6 +12,7 @@ COPY nvme-cli-1.8.1  /nvme-cli-1.8.1
 RUN apt-get update && \
     apt-get -y install open-iscsi \
     sysfsutils \
+    sg3-utils \
```

Comment:
Where are we using this for the code below?

Reply:
> Where are we using this for the code below?

Package needed for Detach() flow at driver side which is used below in newly added code